Source/WebCore/rendering/GridMasonryLayout.h

```diff
@@ -81,9 +80,7 @@ class GridMasonryLayout {
 
     unsigned m_gridAxisTracksCount;
 
-    Vector<RenderBox*> m_itemsWithDefiniteGridAxisPosition;
-    Vector<RenderBox*> m_itemsWithIndefiniteGridAxisPosition;
-
+    Vector<SingleThreadWeakPtr<RenderBox>> m_masonryItems;
```

Comment:
We should see if at some point we can actually get rid of this. Since we have access to the grid iterator in theory we  should be able to use that instead any time we iterate over this Vector

Reply:
Will take a look in a follow up patch.